src/main/resources/application.yml

```diff
@@ -1,6 +1,31 @@
+# http://localhost:8080/swagger-ui/index.html - swagger ui link
 config:
   primary-language: "en"
   languages:
       - "ru"
       - "ua"
       - "en"
+  keepPeriod: 14
+  messageFile: "messages"
+
+spring:
```

Comment:
It is recommended specifying active profile as a program argument and not hardcode it in properties file
or did you mean
```java
spring:
  config:
    activate:
      on-profile: dev

Reply:
Thanks, I will use it as the app argument now